Source/WebKit/Shared/DisplayListArgumentCoders.serialization.in

```diff
@@ -20,17 +20,90 @@
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+headers: <WebCore/DisplayListItem.h>
 headers: <WebCore/DisplayListItems.h>
 
+[CustomHeader, AdditionalEncoder=StreamConnectionEncoder] class WebCore::DisplayList::Save {
+};
+
+[CustomHeader, AdditionalEncoder=StreamConnectionEncoder] class WebCore::DisplayList::Restore {
+};
+
+[CustomHeader, AdditionalEncoder=StreamConnectionEncoder] class WebCore::DisplayList::Translate {
+    float x();
+    float y();
+};
+
+[CustomHeader, AdditionalEncoder=StreamConnectionEncoder] class WebCore::DisplayList::Rotate {
+    float angle();
+};
+
+[CustomHeader, AdditionalEncoder=StreamConnectionEncoder] class WebCore::DisplayList::Scale {
+    WebCore::FloatSize amount();
+};
+
+[CustomHeader, AdditionalEncoder=StreamConnectionEncoder] class WebCore::DisplayList::SetCTM {
+    WebCore::AffineTransform transform();
+};
```

Comment:
This reminds me: if in a future patch we support recording arbitrary display lists, then we'll need to do something like `DrawGlyphsRecorder::updateCTM`, where we effectively convert `setCTM()` calls into equivalent `concatCTM()`s, so that the display list can be replayed at different translations.